app/controllers/api/v1/users/descriptions_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::Users::DescriptionsController < Api::V1::BaseController
+  
+  def create
+    describee = User.find(params[:id])
+    descriptions = JSON.parse(request.body.read)
+    describer    = User.find(descriptions['describer_id'])
+    describee.create_descriptions(descriptions['johari'], describer)
+    if describee.all_descriptions_completed?(describer.id, descriptions['johari'])
+      Assignment.complete(describee.id, describer.id)
```

Comment:
@Dpalazzari @wlffann What if an assignment has already been completed or has not been assigned?

Are we checking to make sure there is an existing assignment for the evaluator and evaluatee and that the assignment has not already been completed before we all the post request to go through?

This seems like the right place to make sure not anyone can submit descriptions for anyone else (or make multiple posts) using something like Postman (unless when we're already doing that and I just can't read code good). 

I'm trying to follow along but thought I'd just ask.